platform-sdk/consensus-otter-tests/docs/container-environment.md

```diff
@@ -459,6 +459,12 @@ docker stats
 docker inspect <container_name>
 ```
 
+### 4. Attaching a Debugger
+
+To attach a debugger to a running container, first identify the node ID of the node you want to attach to. The debug
+port for the container running the consensus node is `5005` plus the node ID (`5005` for node 0, `5006` for node 1,
+etc.). Please note that you cannot attach the debugger until the node has been started.
```

Comment:
What do you think about adding a note that you can make the nodeIds deterministic by setting `@OtterSpecs.randomNodeIds` to `false`?